webapp/src/routes/Home/ShowLifebanks.js

```diff
@@ -189,39 +189,38 @@ const ShowLifebanks = ({ banks, loading, isDesktop }) => {
     )
   }
 
-  const BankItem = (props) => {
-    return (
-      <LinkRouter
-        style={{ textDecoration: 'none' }}
-        to={{
-          pathname: 'info/' + props.bank.info.name.replaceAll(" ", "-"),
-          state: { profile: props.bank }
-        }}
+  const BankItem = (props) => (
+    <LinkRouter
+      style={{ textDecoration: 'none' }}
+      to={{
+        pathname: 'info/' + props.bank.user.username.replaceAll(" ", "-"),
```

Comment:
We can change this string concat (old school XD) and use template string.

example:
```
pathname: `info/${props.bank.user.username.replaceAll(" ", "-")}`
```